.github/workflows/daily.yml

```diff
@@ -19,6 +19,45 @@ on:
         default: 'master'
 
 jobs:
+  tests-with-elle:
```

Comment:
Can you add manual dispatch stuff similar to other jobs?

To select jobs and to prevent running on forks:

```
    if: |
      (github.event_name == 'workflow_dispatch' ||
      (github.event_name == 'schedule' && github.repository == 'redislabs/redisraft')) && !contains(github.event.inputs.skipjobs, 'elle')
```

To set the repo:
```
    - name: prep
      if: github.event_name == 'workflow_dispatch'
      run: |
        echo "GITHUB_REPOSITORY=${{github.event.inputs.use_repo}}" >> $GITHUB_ENV
        echo "GITHUB_HEAD_REF=${{github.event.inputs.use_git_ref}}" >> $GITHUB_ENV
```

Reply:
this I dont quite understand.  need to sit down and dig in.

Reply:
done